applications/ShapeOptimizationApplication/python_scripts/data_logger_factory.py

```diff
@@ -46,6 +46,7 @@ def __init__( self, ModelPartController, Communicator, OptimizationSettings ):
             "optimization_log_filename" : "optimization_log",
             "design_output_mode"        : "WriteOptimizationModelPart",
             "nodal_results"             : [ "SHAPE_CHANGE" ],
+            "write_design_output"       : true,
```

Comment:
I would say there is already an option that could be used for this purpose. Just see the option "design_output_mode" above. You could just introduce another case for this option which is called something like "NoDesignOutput". What do you think?

Reply:
Good Idea :+1:

Reply:
obviously you mean `no_design_output` right :smile:

Reply:
Having a quick look at our json files, i think this is the last remaining setting in ShapeOpt that uses CamelCase naming. I could change it if you agree @dbaumgaertner